test/db/test_db.py

```diff
@@ -77,8 +78,31 @@
     delete_training_template,
     export_job_create,
 )
+from sqlalchemy.ext.asyncio import create_async_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.asyncio import AsyncSession
+
+# Create a unique test directory using absolute paths to prevent contamination
+TEST_BASE_DIR = os.path.abspath(os.path.join(tempfile.gettempdir(), f"transformerlab_db_test_{uuid.uuid4().hex[:8]}"))
```

Comment:
Don't do this here. If we are going to change the way database files are stored for testing, this should be in a different PR (and something we should discuss separately)